spec/controllers/manuals_controller_spec.rb

```diff
@@ -3,11 +3,12 @@
 describe ManualsController, type: :controller do
   describe "#publish" do
     let(:manual_id) { "manual-1" }
+    let(:services) { spy(AbstractManualDocumentServiceRegistry) }
```

Comment:
Do you think it might be worth adding a `context` block around these (while we're here) to indicate that our set up involves a user who doesn't have permission to publish. I'm thinking of something like:

`context "when the user lacks the publish permission" do ... `